main.go

```diff
@@ -167,12 +192,20 @@ func (s *Appender) archiveFileName() string {
 	return s.filePath + "_" + ts
 }
 
+// Append inserts line at the end of file and asks file to be rotated if it is too big.
 func (s *Appender) Append(line string) {
 	if s.bytesWritten >= *maxFileSize {
 		s.rotateFile()
 	}
 	n, _ := s.writer.WriteString(line)
+	if s.syslog != nil && s.regexp != nil {
```

Comment:
why are you putting the syslog part between the file log part? move it down please

Reply:
Already fixed.